.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
I'm probably missing something here but why is this file needed? Can we remove it?